test/general/key.js

```diff
@@ -3301,6 +3301,64 @@ VYGdb3eNlV8CfoEC
       await subKey.verify(privateKey.primaryKey);
     });
 
+    it('create and add a new symmetric subkey to a rsa key', async function() {
+      openpgp.config.tolerant = false;
```

Comment:
(Why do you need this config?)
If you need it, you have to undo the config at the end of the test (and ideally put the entire test in a `try`/`finally` with  `openpgp.config.tolerant = true` in the `finally`).

Reply:
I added this when I was trying to debug the packet parsing. Error would be swallowed silently without it.

I can remove it. The test should fail if anything bad happens, even in tolerant mode.

`Parsing armored text with RSA key and ECC subkey` doesn't reset this either

Reply:
> I can remove it. The test should fail if anything bad happens, even in tolerant mode.

:+1: Btw you can add `&tolerant=false` to the URL when testing in the browser for this purpose; or `&debug=true` to make the warnings show up in the console.

> `Parsing armored text with RSA key and ECC subkey` doesn't reset this either

Oops 😄

Reply:
Should I put in a quick PR to fix that?

Reply:
Go for it if you feel like :+1: I think it can just be removed since `tolerant` is the default